### PR TITLE
Added Directus + Gatsby blog post

### PR DESCRIPTION
**Pull request by alvinometric**
As discussed in #48 here's how to get started with Directus and Gatsby. 

I initially wrote it using just the SDK but then I switched to using the official plugin for a few reasons:
- It misses out on the best feature of Gatsby which is image compression
- Lots of documentation was missing on the plugin page, it took a lot of sleuthing to figure out how to make things work.
- The plugin is the first result on google
- The code is more idiomatic and scales better

### CODE REVIEW

File: gatsby-directus-website/index.md [hunk 1]
```diff
@@ -0,0 +1,290 @@
...
+author:
+  avatar_file_name: alvin-bryan.jpeg
+  name: Alvin Bryan
+---
+
+## Introduction
+
+Hello! This is a blog post to show you how to create a simple static website using Gatsby and Directus as a headless CMS.
+
+We'll make use of the official Directus Gatsby plugin whichs lets hook into the best feature of Gatsby, which is image compression.
```
Comment (phazonoverload) [inline gatsby-directus-website/index.md:13]:
```suggestion
In this blog post, we will create a static website using Gatsby and Directus as a Headless CMS.

We'll make use of the official Directus Gatsby plugin which leverages into one of Gatsby's best features - image compression.
```

Most of this is just for readability, but specifically taking note of the removal of 'simple' https://justsimply.dev/

File: gatsby-directus-website/index.md [hunk 2]
```diff
@@ -0,0 +1,290 @@
...
+
+? Would you like to install additional features with other plugins?
+· Done
+
+❯◯ Add the Google gtag script for e.g. Google Analytics
+· Done
+
+Shall we do this? (Y/n)
+› Yes
+```
```
Comment (phazonoverload) [inline gatsby-directus-website/index.md:57]:
````suggestion
Create a new Gatsby project on your terminal and walk through the wizard:

```sh
npm init gatsby

What would you like to call your site?
  › Directus Gatsby Blog
What would you like to name the folder where your site will be created?
  › directus-gatsby-blog
Will you be using JavaScript or TypeScript?
  › JavaScript
Will you be using a CMS?
  › No (or I'll add it later)
Would you like to install a styling system?
  › No (or I'll add it later)
Would you like to install additional features with other plugins?
  › Done
Add the Google gtag script for e.g. Google Analytics
  › Done
Shall we do this? (Y/n)
  › Yes
```
````

Shrinking it down visually

File: gatsby-directus-website/index.md [hunk 3]
```diff
@@ -0,0 +1,290 @@
...
+npm i gatsby-plugin-image gatsby-transformer-sharp gatsby-plugin-sharp
+```
+
+Start the dev server
+
+```sh
+npm run develop
+```
+
+Your website is live at `http://localhost:8000/`
```
Comment (phazonoverload) [inline gatsby-directus-website/index.md:83]:
````suggestion
Navigate into the new directory and install dependencies:

```sh
cd directus-gatsby-blog
npm install @directus/gatsby-source-directus
npm install gatsby-plugin-image gatsby-transformer-sharp gatsby-plugin-sharp
```

Start the development server

```sh
npm run develop
```

Your website is live at `http://localhost:8000/`
````

It's not too important to explain what every depdendency does - it's ok to just say these are the ones you need now install them.

**Review by phazonoverload — CHANGES_REQUESTED** [review]
I've started a review and then noticed this deviates from the standard format of all of our getting started guides. Please take a look at this and I'll then review the rest. To be clear, the format is: 

- Configure SDK/Package for Directus
- Using Global Metadata and Settings: Directus side then code
- Dynamic Routes With Directus (dynamic arbitrary pages like /about): Directus side then code
- Creating Blog Posts (incl relational authors): Directus side then code
- List Blog Posts
- Add Navigation

**Comment by alvinometric** [comment]
Hey Kevin,
Should I do another blog post ? I'm happy to because I believe this information should be out there as it took me a long time to figure out.

**Comment by phazonoverload** [comment]
But that's what I expected when we spoke originally about this post - I shared the existing framework guides and said the exact same format was what we were after. 

I don't mind if we use the Directus Gastby plugin, and if we talk about using the image compression features as we load images on a blog view, but it's important that the overall format stays the same. 

Let me know how you want to proceed.

**Comment by alvinometric** [comment]
Oh OK, I misunderstood. It's the table of contents (steps) of the article that you'd like revised.
Thank you, I'll update the post.

**Review by phazonoverload — COMMENTED** [review]
Thank you for writing this, and I'm happy to consider it done in it's current state. There are some very small changes I'll make when publishing, but I can handle those. 

This repo is just for authoring/reviewing - the actual piece will be published via a Directus project, so I'll take it from here. I'll close this issue to keep the main branch clean, but this is a acceptance of your work.

Could you email me at [kevin@directus.io](mailto:kevin@directus.io) and I'll share what we need to get you paid.

**Pull request closed without merging**